runtime/queries/rust/highlights.scm

```diff
@@ -156,16 +156,19 @@
   "macro_rules!"
 
   "let"
-  "ref"
-  "move"
 
-  "dyn"
-  "static"
-  "const"
   "async"
 ] @keyword
 
-(mutable_specifier) @keyword.mut
+[
+  ("&" (identifier))
```

Comment:
Checking this out locally, I think I prefer that `&` keeps its `operator` scope when used to take a reference, like in:

```rust
foo(&bar)
```

but I think it could make sense to highlight `&` as storage-class when used as a parameter/type specification:

```rust
pub struct Context<'a> {
    pub jobs: &'a mut Jobs,
}

pub fn foo(&mut self) {}
```

```scm
(reference_type "&" @keyword.storage-class)
(self_parameter "&" @keyword.storage-class)

[
  (mutable_specifier)
  "static"
  "const"
  "ref"
  "move"
  "dyn"
] @keyword.storage-class
```

What do you think?